lib/utils/data_processing_helper.dart

```diff
@@ -0,0 +1,26 @@
+
+
+import 'logging/logger_helper.dart';
+
+// processes event scores data and returns sorted total score for each team in map. Key is teamId, value is total score
+Map<String, int> processScoresData(dynamic scoresData) {
```

Comment:
Zrób może jakąś freezed klase żeby mieć gotowe fromJson() a nie w taki sposob parsować

Reply:
tylko wtedy ten freezed bedzie uzyty w jednym miejscu. To jest zwykle sparsowanie danych by uzyc ich do wyswietlenia. Jezeli jestes przekonany, ze warto zamienic to moge. Daj zac

Reply:
znaczy no to zależy czy to serio jest jedyny use case, może ich przybędzie potem.

Reply:
robienie az tak mega do przodu jest imo zbedne. Jezeli zajdzie taka potrzeba, to wyodrebnimy "zamrozona" klaske XD